react-stopwatch-component/src/index.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+
+class Stopwatch extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      isPaused: true,
+      secondsPassed: 0
+    }
+    this.handleIconClick = this.handleIconClick.bind(this)
+    this.handleWatchClick = this.handleWatchClick.bind(this)
+  }
+  handleIconClick() {
+    if (event.target.className === 'fas fa-play') {
```

Comment:
Try to base your logic on state instead of information you get from DOM elements.